circleci-provision-scripts/scala.sh

```diff
@@ -7,7 +7,7 @@ function install_scala() {
     (cat <<'EOF'
 # Run sbt once to download dependencies.
 # SBT_OPTS="-XX:MaxMetaspaceSize=384M" sbt -v
-SBT_LAUNCH_VERSIONS="0.13.5 0.13.6 0.13.7 0.13.8 0.13.9 0.13.10 0.13.11 0.13.12 0.13.13 0.13.14 0.13.15"
+SBT_LAUNCH_VERSIONS="0.13.5 0.13.6 0.13.7 0.13.8 0.13.9 0.13.10 0.13.11 0.13.12 0.13.13"
```

Comment:
This looks like we're installing only older versions. I don't know much about SBT, but for a newer image it's probably the older ones which should be retired.